chemprop/v2/cli/utils_.py

```diff
@@ -213,14 +214,16 @@ def make_dataset(
     if isinstance(data[0], MoleculeDatapoint):
         extra_atom_fdim = data[0].V_f.shape[1] if data[0].V_f is not None else 0
         extra_bond_fdim = data[0].E_f.shape[1] if data[0].E_f is not None else 0
-        featurizer = MoleculeMolGraphFeaturizer(
+        featurizer = MoleculeMolGraphFeaturizerProto(
```

Comment:
Why do we change `MoleculeMolGraphFeaturizer` into `MoleculeMolGraphFeaturizerProto` here?

Reply:
good catch- will fix